app.py

```diff
@@ -149,10 +149,10 @@ def user_entries(ack, respond, body, command, logger):
     try:
         user_id = body['user_id']
         name = full_name(user_id)
-        num_entries = int(command['text'])
+        num_entries = int(command['text']) if command['text'] != "" else 5
```

Comment:
Well I think it looks better as a default get!
```suggestion
        num_entries = int(command.get('text', 5))
```

Reply:
that is unless it's actually an empty string in the dictionary, then don't do this

Reply:
If the user just does the command with no other text there will still be an empty string in the dictionary so I don't think this works